.github/CODEOWNERS

```
@@ -14,6 +14,7 @@
 /charts/prometheus-cloudwatch-exporter/ @asherf @gianrubio @torstenwalter
 /charts/prometheus-consul-exporter/ @gkarthiks @timm088
 /charts/prometheus-couchdb-exporter/ @gkarthiks
+/charts/prometheus-druid-exporter @iamabhishek-dubey
```

This path needs to end with a trailing slash like the others:
```suggestion
/charts/prometheus-druid-exporter/ @iamabhishek-dubey
```

---

I went ahead and made this change, in an effort to get this out